src/mmw/apps/modeling/tasks.py

```diff
@@ -436,3 +275,98 @@ def to_gms_file(mapshed_data):
     output.seek(0)
 
     return output
+
+
+@shared_task(throws=Exception)
+def analyze_nlcd(result, area_of_interest=None):
+    if 'error' in result:
+        raise Exception('[analyze_nlcd] {}'.format(result['error']))
+
+    pixel_width = aoi_resolution(area_of_interest) if area_of_interest else 1
+
+    histogram = {}
+    total_count = 0
+    categories = []
+
+    # Convert results to histogram, calculate total
+    for key, count in result.iteritems():
+        total_count += count
+        histogram[make_tuple(key[4:])] = count  # Change {"List(1)":5} to {1:5}
```

Comment:
For the soil histogram we also add any values that might already be in the histogram for a given key
(ie`histogram[s] = count + histogram.get(s, 0)`). Is this because there are overlapping results for soil, but not for land?